### PR TITLE
[bazel] Refresh bitstreams cache before copying to airgapped dir

### PR DESCRIPTION
**Pull request by dmcardle**
Signed-off-by: Dan McArdle <dmcardle@google.com>

### CODE REVIEW

**Comment by dmcardle** [comment]
When I ran `util/prep-bazel-airgapped-build.sh`, I saw that the [recursive copy](https://cs.opensource.google/opentitan/opentitan/+/master:util/prep-bazel-airgapped-build.sh;l=169-170;drc=2abd786c664c619dab8a2538c479b3dcf03d9d54) out of the bitstream cache failed because the revision in `latest.txt` had not already been fetched.

I'm guessing this was a regression introduced in #14938 — it looks like we used to just copy whatever we had on disk.

**Review by timothytrippel — APPROVED** [review]

**Comment by dmcardle** [comment]
Thanks, @timothytrippel!

This is a tangent, but would you mind double checking the advice printed at the end of the script?  Shouldn't the `--repository_cache` be defined in terms of `$BAZEL_CACHEDIR` instead of `BAZEL_DISTDIR`?

https://cs.opensource.google/opentitan/opentitan/+/master:util/prep-bazel-airgapped-build.sh;l=181;drc=2abd786c664c619dab8a2538c479b3dcf03d9d54

**Comment by dmcardle** [comment]
Small edit: replaced "tarball" with "dir" in the commit message and PR title.

File: util/prep-bazel-airgapped-build.sh [hunk 1]
```diff
@@ -163,6 +163,9 @@ if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" || \
   readonly SYSTEM_BITSTREAM_CACHE="${HOME}/.cache/opentitan-bitstreams"
   readonly SYSTEM_BITSTREAM_CACHEDIR="${SYSTEM_BITSTREAM_CACHE}/cache"
   readonly LATEST_BISTREAM_HASH_FILE="${SYSTEM_BITSTREAM_CACHE}/latest.txt"
+  # The revision named in latest.txt is not necessarily on disk. Induce the
+  # cache backend to fetch the latest bitstreams.
+  BITSTREAM=latest ${BAZELISK} build --define bitstream=gcp_splice //hw/bitstream:rom
```
Comment (dmcardle) [inline util/prep-bazel-airgapped-build.sh:168]: Oof, just realized I should not have defined `bitstream` here. Force push incoming.

Comment (dmcardle) [reply util/prep-bazel-airgapped-build.sh]: Done.

**Comment by timothytrippel** [comment]
> Thanks, @timothytrippel!
> 
> This is a tangent, but would you mind double checking the advice printed at the end of the script? Shouldn't the `--repository_cache` be defined in terms of `$BAZEL_CACHEDIR` instead of `BAZEL_DISTDIR`?
> 
> https://cs.opensource.google/opentitan/opentitan/+/master:util/prep-bazel-airgapped-build.sh;l=181;drc=2abd786c664c619dab8a2538c479b3dcf03d9d54

Oh yep, good catch @dmcardle !

**Comment by dmcardle** [comment]
Great! Just fixed the echo line in a followup commit.

**Pull request merged**